crates/ruff_linter/src/rules/flake8_implicit_str_concat/rules/implicit.rs

```diff
@@ -81,7 +87,12 @@ impl Violation for SingleLineImplicitStringConcatenation {
 /// ## Options
 /// - `lint.flake8-implicit-str-concat.allow-multiline`
 ///
+/// ## Formatter compatibility
+/// We recommend against using this rule alongside the [formatter]. The
+/// formatter enforces consistent use of implicit string concatenation, making the rule redundant.
+///
 /// [PEP 8]: https://peps.python.org/pep-0008/#maximum-line-length
```

Comment:
Same as for the other implicit-conatenated rule. The formatter doesn't make this rule redundant. We recommend against using this rule with the formatter because the formatter can introduce new violations when it splits an implicit concatenated string over multiple lines. 

However, we address this issue in the new 2025 formatter style guide, at least for as long as a user uses both `SingleLineImplicitStringConcatenation` and `MultiLineImplicitStringConcatenation`

Reply:
I added this compatibility note due to the rule `ISC002` being on the list of incompatible linting rules: https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules.
Perhaps a better message would be:
```rust
/// # Formatter compatibility
/// Use of this rule alongside the [formatter] must be handled with care.
/// Currently, the [formatter] can introduce new multi-line implicitly
/// concatenated strings, therefore we suggest rerunning the linter and
/// [formatter] in the following order:
/// 1. Run the linter with this rule (`ISC002`) disabled
/// 2. Run the [formatter]
/// 3. Rerun the linter with this rule (`ISC002`) enabled
/// This is one of very few cases where the [formatter] can produce code that
/// contains lint violations. It is a known issue that should be resolved by the
/// new 2025 style guide.
```